### PR TITLE
DOC: Fix some incorrect parameters names.

### PR DESCRIPTION
**Pull request by Carreau**
1) interpolation is deprecated and split into interpolation2d/3d.
2) writer's doc seem to not have been updated, and parameter names did
   not match signature

# Description

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change
- [ ] example: I check if my changes works with both PySide and PyQt backends
      as there are small differences between the two Qt bindings.  

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/developers/translations.html).

### CODE REVIEW

File: napari/components/viewer_model.py [hunk 1]
```diff
@@ -665,7 +665,13 @@ def add_image(
...
             expanded as channels.
         interpolation : str or list
-            Interpolation mode used by vispy. Must be one of our supported
+            Deprecated, to be removed in 0.6.0
+        interpolation2d : str or list
+            Interpolation mode used by vispy in D2. Must be one of our supported
+            modes. If a list then must be same length as the axis that is being
+            expanded as channels.
+        interpolation3d : str or list
+            Interpolation mode used by vispy in D2. Must be one of our supported
```
Comment (Czaki) [inline napari/components/viewer_model.py]:
```suggestion
            Interpolation mode used by vispy in 3D. Must be one of our supported
```

File: napari/components/viewer_model.py [hunk 2]
```diff
@@ -665,7 +665,13 @@ def add_image(
             If a list then must be same length as the axis that is being
             expanded as channels.
         interpolation : str or list
-            Interpolation mode used by vispy. Must be one of our supported
+            Deprecated, to be removed in 0.6.0
+        interpolation2d : str or list
+            Interpolation mode used by vispy in D2. Must be one of our supported
```
Comment (Czaki) [inline napari/components/viewer_model.py]:
```suggestion
            Interpolation mode used by vispy in 2D. Must be one of our supported
```

**Review by lucyleeow — APPROVED** [review]
LGTM!

**Review by goanpeca — APPROVED** [review]

File: napari/plugins/_npe2.py [hunk 3]
```diff
@@ -75,15 +75,14 @@ def write_layers(
     ----------
     path : str
         The path (file, directory, url) to write.
-    layer_type : str
+    layers : list of Layers
         All lower-class name of the layer class to be written.
```
Comment (andy-sweet) [inline napari/plugins/_npe2.py]:
```suggestion
        The layers to write.
```

File: napari/plugins/_npe2.py [hunk 4]
```diff
@@ -75,15 +75,14 @@ def write_layers(
...
     plugin_name : str, optional
         Name of the plugin to write data with. If None then all plugins
         corresponding to appropriate hook specification will be looped
         through to find the first one that can write the data.
-    command_id : str, optional
-        npe2 command identifier that uniquely identifies the command to ivoke
-        to save layers. If specified, overrides, the plugin_name.
+    writer : WriterContribution, optional
+        writer contribution to use to write given layer, autodetedect if None.
```
Comment (andy-sweet) [inline napari/plugins/_npe2.py]:
```suggestion
        Writer contribution to use to write given layers, autodetect if None.
```

**Comment by andy-sweet** [comment]
Happy to merge this after a couple of minor suggestions.

**Pull request merged**